packages/core/web-dev-server.config.mjs

```diff
@@ -18,157 +18,49 @@ export default /** @type {import('@web/dev-server').DevServerConfig} */ ({
       inject: {
         importMap: {
           imports: {
-            tslib: '/node_modules/tslib/tslib.es6.js',
+            'lit/': '/node_modules/lit/',
             lit: '/node_modules/lit/index.js',
-            'lit/decorators/custom-element.js': '/node_modules/lit/decorators/custom-element.js',
-            'lit/decorators/property.js': '/node_modules/lit/decorators/property.js',
-            'lit/decorators/query.js': '/node_modules/lit/decorators/query.js',
-            'lit/decorators/state.js': '/node_modules/lit/decorators/state.js',
-            'lit/directives/if-defined.js': '/node_modules/lit/directives/if-defined.js',
-            'lit/directives/unsafe-html.js': '/node_modules/lit/directives/unsafe-html.js',
-            'lit/directive-helpers.js': '/node_modules/lit/directive-helpers.js',
-            'ramda/es/anyPass.js': '/node_modules/ramda/es/anyPass.js',
-            'ramda/es/clone.js': '/node_modules/ramda/es/clone.js',
-            'ramda/es/equals.js': '/node_modules/ramda/es/equals.js',
-            'ramda/es/isNil.js': '/node_modules/ramda/es/isNil.js',
-            'ramda/es/includes.js': '/node_modules/ramda/es/includes.js',
-            'ramda/es/curryN.js': '/node_modules/ramda/es/curryN.js',
-            'ramda/es/path.js': '/node_modules/ramda/es/path.js',
-            'ramda/es/has.js': '/node_modules/ramda/es/has.js',
-            'ramda/es/is.js': '/node_modules/ramda/es/is.js',
-            'ramda/es/isEmpty.js': '/node_modules/ramda/es/isEmpty.js',
-            'ramda/es/without.js': '/node_modules/ramda/es/without.js',
-
-            '@cds/core/global.min.css': '/dist/core/global.min.css',
-            '@cds/core/styles/theme.dark.min.css': '/dist/core/styles/theme.dark.min.css',
-            '@cds/core/styles/theme.low-motion.min.css': '/dist/core/styles/theme.low-motion.min.css',
-            '@cds/core/list/list.min.css': '/dist/core/list/list.min.css',
-            '@cds/core/table/table.min.css': '/dist/core/table/table.min.css',
-            '@cds/core/tokens/tokens.json': '/dist/core/tokens/tokens.json',
-
+            'ramda/': '/node_modules/ramda/',
+            ramda: '/node_modules/ramda/es/index.js',
+            'tslib/': '/node_modules/tslib/',
+            tslib: '/node_modules/tslib/tslib.es6.js',
+            '@cds/core/': '/dist/core/',
```

Comment:
The import map can be simplified now since we add `.js` to all imports except top level module entrypoints